Tasks/Common/webdeployment-common/package.json

```diff
@@ -15,9 +15,8 @@
   "dependencies": {
     "archiver": "1.2.0",
     "decompress-zip": "0.3.0",
-    "ltx": "2.6.2",
+    "ltx": "2.5.0",
```

Comment:
why is this changed?

Reply:
current build of the task from master bundles ltx 2.5.0, and not 2.6.2 . Not sure why it was there in the first place.

Reply:
found the reason, 2.5.0 is the version that is mentioned in shrinkwrap. whereas 2.6.2 is the one mentioned in package.json. and shrinkwrap takes precedence